src/test/java/ScattTest.java

```diff
@@ -123,6 +123,39 @@ public File getDirectoryFromUser() {
             fail("You need to put a file called " + testDir.getName() + "_Report.txt containing" 
                  + " the expected report contents in the test resources directory");
         }
+        if (!actual.equals(expected)) {
```

Comment:
I'm tempted to pull in a library to do this kind of diff-ing. It's pretty complex stuff. Thoughts?